apps/web/app/page.tsx

```diff
@@ -1,15 +1,19 @@
 // 3rd party
 import { JSX } from "react";
 // repo
-import Link from "@repo/ui/link";
-import { URL_WALKTHROUGH_HREF } from "@repo/constants/src/urls";
+import WalkthroughCard from "@repo/ui/walkthrough-card";
+import { WalkthroughJSONData } from "@repo/data/useWalkthroughData";
 
 export default function Page(): JSX.Element {
   return (
     <div className="u-container">
-      <p>
-        <Link href={`${URL_WALKTHROUGH_HREF}/9.9.9/`}>Walkthrough 9.9.9</Link>
-      </p>
+      <div>
+        {Object.entries(WalkthroughJSONData).map(([id, { info }]) => {
+          return (
+            <WalkthroughCard key={id} id={id} data={info} walkthroughId={id} />
```

Comment:
Also now since `id` and `walkthroughId` are the same I'm not sure we need separate props for that.